acbuild/add-anno.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/appc/acbuild/libacb"
+)
+
+var (
+	cmdAddAnno = &cobra.Command{
+		Use:     "add-anno NAME VALUE",
```

Comment:
About `command object` vs `object command` I don't have a strong opinion. Given that the command that works on objects are just `add` and `rm` while the objects can be a a lot it'll be more easy to group the objects under them...
With cobra it's also easy to add subcommands.